examples/dt/dcn/service-values.yaml

```diff
@@ -0,0 +1,246 @@
+          rbd_store_user = openstack
+          rbd_thin_provisioning = True
+        networkAttachments:
+        - storage
+        override:
+          service:
+            internal:
+              metadata:
+                annotations:
+                  metallb.universe.tf/address-pool: internalapi
+                  metallb.universe.tf/allow-shared-ip: internalapi
+                  metallb.universe.tf/loadBalancerIPs: 172.17.0.81
+              spec:
+                type: LoadBalancer
+        replicas: 3
+        type: edge
+      az2:
+        customServiceConfig: |
+          [DEFAULT]
+          enabled_import_methods = [web-download,copy-image,glance-direct]
+          enabled_backends = ceph:rbd,az1:rbd,az2:rbd
+          [glance_store]
+          default_backend = az2
+          [az2]
+          rbd_store_ceph_conf = /etc/ceph/az2.conf
+          store_description = "az2 RBD backend"
+          rbd_store_pool = images
+          rbd_store_user = openstack
+          rbd_thin_provisioning = True
+          [ceph]
+          rbd_store_ceph_conf = /etc/ceph/ceph.conf
+          store_description = "ceph RBD backend"
+          rbd_store_pool = images
+          rbd_store_user = openstack
+          rbd_thin_provisioning = True
+          [az1]
+          rbd_store_ceph_conf = /etc/ceph/az1.conf
+          store_description = "az1 RBD backend"
+          rbd_store_pool = images
+          rbd_store_user = openstack
+          rbd_thin_provisioning = True
+        networkAttachments:
+        - storage
+        override:
+          service:
+            internal:
+              metadata:
+                annotations:
+                  metallb.universe.tf/address-pool: internalapi
+                  metallb.universe.tf/allow-shared-ip: internalapi
+                  metallb.universe.tf/loadBalancerIPs: 172.17.0.82
+              spec:
+                type: LoadBalancer
+        replicas: 3
+        type: edge
+  manila:
+    enabled: true
+    manilaAPI:
+      customServiceConfig: |
+        [DEFAULT]
+        enabled_share_protocols=nfs,cephfs
+    manilaShares:
+      share1:
+        customServiceConfig: |
+          [DEFAULT]
+          enabled_share_backends = cephfs
+          enabled_share_protocols = cephfs
+          [cephfs]
+          driver_handles_share_servers = False
+          share_backend_name = cephfs
+          share_driver = manila.share.drivers.cephfs.driver.CephFSDriver
+          cephfs_conf_path = /etc/ceph/ceph.conf
+          cephfs_cluster_name = ceph
+          cephfs_auth_id=openstack
+          cephfs_volume_mode = 0755
+          cephfs_protocol_helper_type = CEPHFS
+  neutron:
+      template:
+          customServiceConfig: |
+            [ml2_type_vlan]
+            network_vlan_ranges = datacentre:1:1000,leaf1:1:1000,leaf2:1:1000
+            [neutron]
+            physnets = datacentre,leaf1,leaf2
+  nova:
+      customServiceConfig: |
+        [DEFAULT]
+        default_schedule_zone=az-dcn0
```

Comment:
I can't see the default AZ set anywhere on deployed env and default AZ for nova should be az0 If It's done the way (to be consistent with other parts of generated CRs). This should imo be:
```
nova:
    template:
      apiServiceTemplate:
        customServiceConfig: |
          [DEFAULT]
          default_schedule_zone = az0
```

Reply:
Done